.github/agents/copilot-instructions.md

```diff
@@ -0,0 +1,31 @@
+# mac-test-reliability-fixes Development Guidelines
+
```

Comment:
I'm not planning to review this PR any time soon since it's enormous and the outcome isn't high priority, but please let me know if you think it should be.